provider/aws/aws.go

```diff
@@ -134,6 +158,69 @@ type Route53API interface {
 	ListTagsForResourceWithContext(ctx context.Context, input *route53.ListTagsForResourceInput, opts ...request.Option) (*route53.ListTagsForResourceOutput, error)
 }
 
+type RateLimitedRoute53Api struct {
```

Comment:
```suggestion
type RateLimitedRoute53API struct {
```

so it's consistent with the rest of the things like `type Route53API`